src/pages/en/guides/images.md

```diff
@@ -142,7 +141,9 @@ You must also either provide `width` and `height`, or one of the dimensions plus
 
 #### Local Images in `public/`
 
-The `<Image />` component can also be used with images stored in the `public/` directory and the `src` attribute is relative to the public folder. It will be treated as a remote image, which requires either both `width` and `height`, or one dimension and an `aspectRatio` attribute.
+The `<Image />` component can also be used with images stored in the `public/` directory and the `src` attribute is relative to the public folder. It will be treated as a remote image, which requires either both `width` and `height`, or one dimension and an `aspectRatio` attribute. 
+
+Note that your original image will be copied unprocessed to the build folder, like all files located in `public/`. However, Astro's image integration is still able to return optimized versions of your images stored in `public/`.
```

Comment:
Style nit + is it copied BUT or copied AND?

```suggestion
Your original image will be copied unprocessed to the build folder, like all files located in `public/`, and Astro's image integration will also return optimized versions of the image.
```

Reply:
I like what you wrote better, but there's a difference between SSG and SSR  😅 , so I just want to make sure this still is accurate.

In SSR, only the original image is copied over, but the fetch call returns an optimized image (without creating a second image in your build folder). In SSG, you end up with TWO copies of your image copied into the build folder: the original, and an optimized one.

I think that's why I said "is able to return" which admittedly sounds clunky. I think "will also return" is legit here, eh?

Reply:
Yeah, I think “will also return” is ok! There’s nothing in here that really explains any of that SSR/SSG distinction and “return” is sufficiently vague as to not mean “copy to `dist`”, I think, so would apply in all scenarios.